app/webpacker/components/Tickets/TicketWorkbenches/EditPersonTicketWorkbench.jsx

```diff
@@ -26,6 +26,28 @@ function EditPersonValidations({ ticketDetails }) {
   ));
 }
 
+function EditPersonRequestedChangesList({ ticketDetails }) {
+  const { ticket } = ticketDetails;
+  const { tickets_edit_person_fields: requestedChanges } = ticket.metadata;
```

Comment:
If `ticketDetails` is used only for `requestedChanges`, then maybe you can pass that alone to this class.